binary_search_tree/binary_search_tree.py

```diff
@@ -5,13 +5,38 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    if value < self.value:
+      if not self.left:
+        self.left = BinarySearchTree(value)
+      else:
+        self.left.insert(value)
+    elif value >= self.value:
+      if not self.right:
+        self.right = BinarySearchTree(value)
+      else:
+        self.right.insert(value)
 
   def contains(self, target):
-    pass
+    if self.value == target:
+      return True
+    elif self.left is None and self.right is None:
+      return False
+    elif target < self.value:
+      return self.left.contains(target)
+    else:
+      return self.right.contains(target)
 
```

Comment:
Nice work utilizing the structure of BST to create efficient operations for contains/getMax.